_rules/SC2-4-6-descriptive-headings.md

```diff
@@ -22,13 +23,13 @@ authors:
 
 ### Applicability
 
-This rule applies to any element with the [semantic role](#semantic-role) of heading that is either [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree).
+This rule applies to any element with the [semantic role](#semantic-role) of heading that is either [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree), and that either contains text nodes that do not only consist of [Unicode separator characters](https://www.unicode.org/versions/Unicode11.0.0/ch04.pdf#G134153), or has an accessible name that does not only consist of Unicode separator characters.
```

Comment:
Suggestion:

...and does not only consist of [Unicode separator characters]...